packages/terra-spacer/tests/nightwatch/spacer-spec.js

```diff
@@ -0,0 +1,263 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+/* eslint-disable no-unused-expressions */
+const resizeTo = require('terra-toolkit/lib/nightwatch/responsive-helpers').resizeTo;
+
+module.exports = resizeTo(['tiny', 'small', 'medium', 'large', 'huge', 'enormous'], {
+  'Displays a default spacer': (browser) => {
+    browser
+      .url(`${browser.launchUrl}/#/tests/spacer-tests/default`)
+      .expect.element('#spacer').to.be.present;
+  },
+
+  'Displays spacer with very very small margin': (browser) => {
+    browser
+      .url(`${browser.launchUrl}/#/tests/spacer-tests/SpacerSmall2`)
+      .expect.element('#spacermarginsmall-2').to.be.present;
+    browser.expect.element('#spacerpaddingsmall-2').to.be.present;
+    browser.expect.element('#spacermarginpaddingsmall-2').to.be.present;
```

Comment:
could add a .to.equal(true) to all of these expectations that the element is to.be.present
then could remove the eslint-disable no-unused-expressions at the top

ex.
```
browser.expect.element('#spacermarginpaddingsmall-2').to.be.present.to.equal(true);

Reply:
Let's leave this as is.